src/index.css

```diff
@@ -72,6 +72,11 @@ svg.marker text {
   font-weight: 500;
 }
 
+.daily-growth-filter {
+  display: flex;
+  justify-content: space-evenly;
+}
```

Comment:
It should be removed as stated in [a comment above](https://github.com/konradkalemba/korona.ws/pull/75/files#r419251881).
```suggestion
```